project_task_delegate/wizards/project_task_delegate.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from lxml import etree
+
+from openerp import tools
+from openerp.tools.translate import _
+from openerp import models, fields, api
+
+
+class ProjectTaskDelegate(models.TransientModel):
+    _name = 'project.task.delegate'
+    _description = 'Task Delegate'
+
+    name = fields.Char('Delegated Title', required=True,
+                       help="New title of the task delegated to the user")
+    prefix = fields.Char(
+        'Your Task Title', help="Title for your validation task")
+    project_id = fields.Many2one(
+        'project.project',
+        'Project',
+        help="User you want to delegate this task to")
+    user_id = fields.Many2one('res.users', 'Assign To', required=True,
+                              help="User you want to delegate this task to")
+    new_task_description = fields.Html(
+        'New Task Description',
+        help="Reinclude the description of the task in the task of the user")
+    planned_hours = fields.Float(
+        'Planned Hours',
+        default=1.0,
+        help="Estimated time to close this task by the delegated user")
+    planned_hours_me = fields.Float(
+        'Hours to Validate',
+        help="""Estimated time for you to validate the work done
+             by the user to whom you delegate this task""")
+    state = fields.Selection([('pending', 'Pending'), ('done', 'Done'), ],
+                             'Validation State', default='pending',
+                             help="""New state of your own task. Pending will be
+             reopened automatically when the delegated task is closed""")
+
+    @api.onchange('project_id')
+    def onchange_project_id(self):
+        self.user_id = self.project_id.user_id.id
+
+    @api.model
+    def default_get(self, fields):
+        """
+        This function gets default values
+        """
+        res = super(ProjectTaskDelegate, self).default_get(fields)
+        context = dict(self._context or {})
+        record_id = context.get('active_ids', False)
+        task = self.env['project.task'].browse(record_id)
+        task_name = tools.ustr(task.name)
+
+        if 'project_id' in fields:
+            res['project_id'] = int(
+                task.project_id.id) if task.project_id else False
+
+        if 'name' in fields:
+            if task_name.startswith(_('CHECK: ')):
+                newname = tools.ustr(task_name).replace(_('CHECK: '), '')
```

Comment:
[R74](07e1bb2823e91a4b658a5bdda87b682c609a5119#diff-df61f31c4ab01ba08a21ebf274c6dec5R74)

```
        if 'prefix' in fields:
            if task_name.startswith(_('CHECK: ')):
                newname = tools.ustr(task_name).replace(_('CHECK: '), '')
            else:
                newname = tools.ustr(task_name or '')
            prefix = _('CHECK: %s') % newname
            res['prefix'] = prefix
```

Here we are adding the constant `CHECK: ` to the delegation wizard's `default_get`. 
That constant will be written to the parent task if unchanged by the user while doing delegation.